### PR TITLE
chore: refactor pr overview presentation on repo tables

### LINKED ISSUE #2326: Change the presentation of the PR Overview % in lists
## Current
Right now, if you look at the PR Overview %, it defaults to red. This is misleading, because without looking at the information, it appears that every repo is in the red. 

image

## Suggested Solution

I think there's a couple of things that should happen here:

1. Remove the up and down arrow, since they don't really correspond to the data being presented.
2. Make the default number the largest number on the line. If this is too complicated, then I think we should default to merged PRs (purple).

image

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR changes the presentation of `pull request overview chart` on the repositories table

 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
closes #2326 

## Mobile & Desktop Screenshots/Recordings
https://github.com/open-sauced/app/assets/62995161/a72723b3-197b-4149-8b8f-53d3d6877fb1

## Steps to QA

1. Goto explore page
2. scroll to the repositories table and hover on different pr overview column 

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/molecules/PullRequestOverview/pull-request-overview.tsx [hunk 1]
```diff
@@ -12,58 +11,33 @@ interface PullRequestOverviewProps {
...
-  const prCount = prActiveCount || 0;
-  const activePrPercentage = totalPullRequests > 0 ? Math.round((prCount / totalPullRequests) * 100) : 0;
-  // used this state to manage actively hovered pull request overview chart
-  interface OverviewDetails {
-    type: string;
-    percent: number | undefined;
-  }
 
-  const [prOverviewDetails, setPrOverviewDetails] = useState<OverviewDetails>({ type: "closed", percent: closed });
+  // used this state to manage actively hovered pull request overview chart
```
Comment (nickytonline) [inline components/molecules/PullRequestOverview/pull-request-overview.tsx:24]: nit: the comment isn't necessary

File: components/molecules/PullRequestOverview/pull-request-overview.tsx [hunk 2]
```diff
@@ -12,58 +11,33 @@ interface PullRequestOverviewProps {
...
+  const [prOverviewDetails, setPrOverviewDetails] = useState<OverviewDetails>({
+    type: "",
```
Comment (nickytonline) [inline components/molecules/PullRequestOverview/pull-request-overview.tsx:26]: should the type be an empty string by default? The types appear to be "open" "closed" and "merged" only.

File: components/molecules/PullRequestOverview/pull-request-overview.tsx [hunk 3]
```diff
@@ -12,58 +11,33 @@ interface PullRequestOverviewProps {
   churnDirection?: string;
   prActiveCount?: number;
 }
+interface OverviewDetails {
+  type: string;
+  percent: number | undefined;
+}
```
Comment (nickytonline) [inline components/molecules/PullRequestOverview/pull-request-overview.tsx:17]: Refactor (blocking): Narrow the type for `type`. `"open"`, `"closed"`, and `"merged"` seemed to be the only types allowed. Maybe an empty string as well? See my comment below.

```suggestion
interface OverviewDetails {
  type: "open" | "closed" | "merged";
  percent: number | undefined;
}
```

**Review by nickytonline — CHANGES_REQUESTED** [review]

File: components/molecules/PullRequestOverview/pull-request-overview.tsx (hunk 2 above)
Comment (OgDev-01) [reply components/molecules/PullRequestOverview/pull-request-overview.tsx:26]: The default state is the total prs... `merged` + `open` + `closed` + `draft`. And that doesn't require a type prefix. I'll adjust the type to be explicitly defined as suggested with an empty string to it.

**Comment by brandonroberts** [comment]
Closing as we'll revisit this overview soon

**Pull request closed without merging**